src/test/java/net/mcreator/integration/generator/GTModElements.java

```diff
@@ -96,7 +96,7 @@ public static void runTest(Logger LOG, String generatorName, Random random, Work
 				for (File modElementFile : modElementFiles) {
 					ModElement modElement1 = workspace.getGenerator().getModElementThisFileBelongsTo(modElementFile);
 					if (!modElement.equals(modElement1))
-						fail("Filed to properly determine file ownership for mod element type: " + modElement.getType()
+						fail("Filed to properly determine file ownership for mod element type: " + modElement.getName()

```

Comment:
Why did you replace getType with getName here?

Reply:
Oops, this is due to the refractoring.